### PR TITLE
Switch from Uglify to Terser

### PR DESCRIPTION
**Pull request by poi33**
css url() was not working properly with the default css-loader configuration

### CODE REVIEW

**Comment by poi33** [comment]
connected to #105

File: webpack.config.js [hunk 1]
```diff
@@ -26,7 +25,7 @@ JS_CONFIG = {
         filename: "assets/js/[name].min.js",
     },
     optimization: {
-        minimizer: [new UglifyJsPlugin()],
+        minimize: true,
```
Comment (alansemenov) [inline webpack.config.js:28]: Hmm, but you are not actually using TerserPlugin for minimizing?

```Javascript
    optimization: {
        minimizer: [
            new TerserPlugin({
                terserOptions: {
                    compress: {
                        drop_console: false
                    },
                    keep_classnames: true,
                    keep_fnames: true
                }
            })
        ]
    },
```

**Review by alansemenov — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 1 above)
Comment (poi33) [reply webpack.config.js:28]: Webpack v5 comes with the latest terser-webpack-plugin out of the box. If you are using Webpack v5 or above and wish to customize the options, you will still need to install terser-webpack-plugin.

Comment (poi33) [reply webpack.config.js:28]: I did not see any configuration of the UglifyPlugin. Are the options needed?

**Comment by alansemenov** [comment]
[Example](https://webpack.js.org/plugins/terser-webpack-plugin/) in the Webpack docs is explicitly using TerserPlugin. So maybe we don't need it in `package.json` (since it comes out of the box) but we definitely need it in `webpack.config`.

 ```Javascript
 optimization: {
    minimize: true,
    minimizer: [new TerserPlugin()],
  },
```

**Comment by poi33** [comment]
Using the terser plugin like in the example you wrote is functionally the same as the pull request.

**Review by alansemenov — APPROVED** [review]

**Pull request merged**